_events/sr2021/league-2.md

```diff
@@ -1,10 +1,22 @@
 ---
 title: Competition League Session 2
-date: 2021-02-06
-time_tbc: true
+date: 2021-02-06 12:00:00 Europe/London
 layout: event
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=RwW5Oz30gbE">Student Robotics' YouTube Channel</a>
 ---
 
 This is the second league round of this year's competition and we'll get to see what lessons teams have learnt from their first time in the arena! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm GMT on Friday 5th: [Submit your code](https://studentrobotics.org/code-submitter/)
+
+Our live-stream will start at noon GMT: [Watch live](https://www.youtube.com/watch?v=RwW5Oz30gbE)
```

Comment:
I've been trying to reword this to make it more casual (this was the best I got)
```suggestion
We'll be kicking things off at noon GMT, be sure to [watch us live](https://www.youtube.com/watch?v=RwW5Oz30gbE)!
```
but then I was wondering if we need this sentence at all? Both the time and the stream (embedded) will already be on the page?